src/libraries/System.Security.Permissions/src/System/Data/Common/DBDataPermission.cs

```diff
@@ -12,20 +12,20 @@ namespace System.Data.Common
     public abstract class DBDataPermission : CodeAccessPermission, IUnrestrictedPermission
     {
         protected DBDataPermission() { }
-        protected DBDataPermission(DBDataPermission dataPermission) { }
-        protected DBDataPermission(DBDataPermissionAttribute attribute) { }
+        protected DBDataPermission(DBDataPermission permission) { }
+        protected DBDataPermission(DBDataPermissionAttribute permissionAttribute) { }
```

Comment:
You had a comment that you generally went with changing `ref` and preserving `src`, but this does the opposite.  I agree, but I'm guessing that the exceptions to that general statement are

* If ref agrees with NetFX, ref wins (this case)
* If no exceptions mention the parameter by name, and the member has always had a ref assembly representation, ref wins (the Rfc3161* types case)

I'm good with both classes, just seeing if my mental model matches reality.

Reply:
I tried to do a bit of digging in history for most cases to select the best option.  Here it looks like this one was added and it was out of sync with ref from the get-go and the ref matched desktop.  Given this is a compat assembly and its better to match desktop I chose to do so here.